requirements/py38.yml

```diff
@@ -14,6 +14,7 @@ dependencies:
   - numpy>=1.14
   - scipy
   - esmpy>=7.0
+  - iris
```

Comment:
The alternate version of iris is just for the feature branch. The reason the feature branch is still a feature branch is because it depends on an unreleased  version of iris. This scheme only works with non-mesh cubes so it can go on main and use the normal versio of iris.